mmv1/third_party/terraform/services/compute/resource_compute_instance_group_manager.go.erb

```diff
@@ -216,6 +216,49 @@ func ResourceComputeInstanceGroupManager() *schema.Resource {
 				},
 			},
 
+			<% unless version == "ga" -%>
+			"standby_policy": {
+				Computed:    true,
+				Type:        schema.TypeList,
+				Optional:    true,
+				MaxItems:    1,
+				Description: `Standby policy for stopped and suspended instances.`,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"initial_delay_sec": {
+							Type:             schema.TypeInt,
+							Default:          0,
```

Comment:
If no value is provided in the API request for this field, will the API side default it to 0 and send it back in the API response?

Normally, we don't recommend to set the default value in Terraform provider, in case the default value changes later in the server side.

Reply:
During instance_group_manager creation, yes. It will set to 0 default. For update no. It will return the previous value which was set on it.